lib/main_common.pm

```diff
@@ -1058,7 +1058,7 @@ sub load_consoletests {
     loadtest "console/prepare_test_data";
     loadtest "console/consoletest_setup";
     loadtest 'console/integration_services' if is_hyperv || is_vmware;
-    loadtest "locale/keymap_or_locale";
+    loadtest "locale/keymap_or_locale" if check_var('DESKTOP', 'textmode');
```

Comment:
hm, that would be about the only test module that we would handle specially within the console tests in this section. I would prefer if we either just execute it regardless, i.e. also in the "non-textmode" scenarios same as the other test modules here or move it to https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6634/files#diff-d3e0a6c7df772cff7741832a6250d5afR1170

Reply:
I mean the exclusion for non-textmode we only have for that module

Reply:
+1 for run it unconditionally